doc/source/user/absolute_beginners.rst

```diff
@@ -0,0 +1,2451 @@
+
+**Input:**
+
+::
+
+  five_up = (array > 5) | (array == 5)
+  print(five_up)
+
+**Output:**
+
+::
+
+  [[False False False False]
+   [ True  True  True  True]
+   [ True  True  True  True]] 
+
+You can also use ``np.where()`` to select elements or indices from an array. 
+
+Starting with this array:
+
+::
+
+  a = np.array([[1 , 2, 3, 4], [5, 6, 7, 8], [9, 10, 11, 12]])
+
+You can use ``np.where()`` to print the indices of elements that are, for example, less than 5:
+
+**Input:**
+
+::
+
+  b = np.where(a<5)
+  print(b)
+
+**Output:**
+
+::
+
+  (array([0, 0, 0, 0]), array([0, 1, 2, 3]))
+
+In this example, a tuple of arrays was returned: one for each dimension. The first array represents the row indices where these values are found, and the second array represents the column indices where the values are found.
+
+If you want to generate a list of coordinates where the elements exist, you can zip the arrays, iterate over the list of coordinates, and print them. For example:
+
+**Input:**
+
+::
+
+  list_of_coordinates= list(zip(b[0], b[1]))
+
+  for cord in list_of_coordinates:
+      print(cord)
+
+**Output:**
+
+::
+
+  (0, 0)
+  (0, 1)
+  (0, 2)
+  (0, 3)
+
+You can also use ``np.where()`` to print the elements in an array that are less than 5 with:
+
+**Input:**
+
+::
+
+  print(a[b])
+
+**Output:**
+
+::
+
+  [1 2 3 4]
+
+If the element you're looking for doesn't exist in the array, then the returned array of indices will be empty. For example:
+
+**Input:**
+
+::
+
+  not_there = np.where(a == 42)
+  print(not_there)
+
+**Output:**
+
+::
+
+  (array([], dtype=int64), array([], dtype=int64))
+
+
+`Learn more about indexing and slicing here <https://docs.scipy.org/doc/numpy-1.17.0/user/quickstart.html#indexing-slicing-and-iterating>`_ and `here <https://docs.scipy.org/doc/numpy-1.17.0/user/basics.indexing.html>`_.
+
+`Read more about using the where function here <https://docs.scipy.org/doc/numpy-1.15.1/reference/generated/numpy.where.html>`_.
+
+
+How to create an array from existing data
+-----------------------------------------
+
+
+::
+
+  slicing and indexing
+
+  np.vstack()
+  np.hstack()
+  np.hsplit()
+  
+  .view()
+  .copy()
+
+-----
+
+You can easily use create a new array from a section of an existing array. 
+
+Let's say you have this array:
+
+::
+
+  array([ 1,  2,  3,  4,  5,  6,  7,  8,  9, 10])
+
+You can create a new array from a section of your array any time by specifying where you want to slice your array.
+
+**Input:**
+
+::
+
+  arr1 = np.array[3:8]
+  arr1
+
+**Output:**
+
+::
+
+  array([4, 5, 6, 7, 8])
+
+Here, you grabbed a section of your array from index position 3 through index position 8.
+
+You can also stack two existing arrays, both vertically and horizontally. Let's say you have two arrays: 
+
+**a_1**:
+
+::
+
+  array([[1, 1],
+       [2, 2]])
+
+and **a_2**:
+
+::
+
+  array([[3, 3],
+       [4, 4]])
```

Comment:
this doesn't line up, missing two spaces before `[4`

Reply:
Got it!